lstmcpipe/scripts/script_batch_filelist_lst.py

```diff
@@ -53,7 +53,9 @@ def main():
             if args.config_file:
                 cmd.append("--config={}".format(args.config_file))
 
-            subprocess.run(cmd)
+            file = Path(file)
```

Comment:
Just do `type=Path` in the argumentparse definition above.

Reply:
`file` is read from the filelist file, not passed through argparse